bbc/simorgh
# Korean Top stories heading fix plus updates to fr, bn, fa navigation

Resolves JIRA [number]

Overall changes
======
Fixed Korean's Top stories heading. Updated to the navigation of Afrique to merge Science and Technology into one single topic, the navigation of Bengali to add link to Elections 2024 topic plus rearranging the list's order and to Persian nav to remove the Blogs link.

Code changes
======

- edited topStoriesTitle on src/app/lib/config/services/korean.ts
- edited the navigation section of src/app/lib/config/services/afrique.ts to add link to /afrique/topics/crezq2zk0q4t and remove links to Science and Technologie
- edited the navigation section of src/app/lib/config/services/bengali.ts to add link to /bengali/topics/c90xlq1n7llt
- edited the navigation of src/app/lib/config/services/persian.ts to remove the link to /persian/blogs-54099951
- updated snapshots

Testing
======
1. Open a Korean article (for instance http://localhost:7080/korean/articles/c3mn1lvz65xo) the Top stories heading should be 주요 뉴스 wiht a space between the two words as opposed to 주요뉴스 without space as it appears currently on live)
2. Open Afrique's front page the sixth item in the navigation should be 'Science et tecnologie' pointing to /afrique/topics/crezq2zk0q4t. 
3. Open the Bengali front page the second item in the navigation should be সংসদ নির্বাচন ২০২৪ (/bengali/topics/c90xlq1n7llt), the navigation items should follow this order: Home| Election 2024 | Most read | World | Economy | Health | Sport | Technology | Politics | Videos
4. Open the Persian front page, the Blogs link 'ناظران می‌گویند' (/persian/blogs-54099951) should no longer be in second to last position

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
